packages/nimble-components/src/wafer-map/tests/computations.spec.ts

```diff
@@ -6,87 +7,95 @@ describe('Wafermap Computations module', () => {
     let computationsModule: Computations;
 
     describe('with 100 square canvas', () => {
-        const defaultMargin: Margin = {
-            top: 20,
-            right: 20,
-            bottom: 20,
-            left: 20
+        const expectedMargin: Margin = {
+            top: 4,
+            right: 4,
+            bottom: 4,
+            left: 4
         };
         beforeEach(() => {
-            computationsModule = new Computations(
-                getWaferMapDies(),
-                WaferMapQuadrant.topLeft,
-                { width: 100, height: 100 }
-            );
+            computationsModule = new Computations({
+                dies: getWaferMapDies(),
+                quadrant: WaferMapQuadrant.topLeft,
+                canvas: { width: 100, height: 100 }
+            } as WaferMap);
```

Comment:
A safer way to mock an object is by using the [Pick utility type](https://www.typescriptlang.org/docs/handbook/utility-types.html#picktype-keys). This will help make sure the data types on the mock are accurate and help during refactors, etc. by making sure that the properties match the original object type.

For example:

```suggestion
            const canvasMock: Pick<HTMLCanvasElement, 'width' | 'height'> = { width: 100, height: 100 };
            const waferMock: Pick<WaferMap, 'dies' | 'quadrant' | 'canvas'> = {
                dies: getWaferMapDies(),
                quadrant: WaferMapQuadrant.topLeft,
                canvas: canvasMock as HTMLCanvasElement
            };
            computationsModule = new Computations(waferMock as WaferMap);
```

The above will be simplified a bit when you no longer need to mock the canvas object.